packages/validator/src/services/attestationDuties.ts

```diff
@@ -168,7 +168,7 @@ export class AttestationDutiesService {
     for (const epoch of [currentEpoch, nextEpoch]) {
       // Download the duties and update the duties for the current and next epoch.
       await this.pollBeaconAttestersForEpoch(epoch, indexArr).catch((e: Error) => {
-        this.logger.error("Failed to download attester duties", {epoch}, e);
+        this.logger.warn("Failed to download attester duties", {epoch}, e);
```

Comment:
Reference: https://github.com/ChainSafe/lodestar/issues/5359#issuecomment-1529027981
Not sure if this should be an `error`? We also should suppress it if it's triggered due to `Node is syncing` state.

Reply:
This is a broader issue around handling bad beacon node connections more gracefully. This make sense to be an error because it's critical, AND the user can do something about it (say if the beacon node is down or unsynced). However the problem is the possible spam of errors

Reply:
would agree that `error` here is the correct log level, I opened a separate PR to fix this log if node is syncing
- https://github.com/ChainSafe/lodestar/pull/5467